### PR TITLE
do it

### PR DESCRIPTION
**Pull request by BoogieMonster1O1**
yes
image

- updated yarn
- stopped compiling stuff
- added a config api as a dependency
- did some more stuff

### CODE REVIEW

**Comment by hYdos** [comment]
we already have a config mod. use it :) it avoids so much of that config code you put in

File: src/main/resources/tabcontrol.mixins.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "package": "io.github.hydos.tabcontrol.mixin",
   "compatibilityLevel": "JAVA_8",
   "server": [
+    "PlayerListHeaderS2CPacketAccessor"
```
Comment (hYdos) [inline src/main/resources/tabcontrol.mixins.json:7]: access widener or bad

**Review by hYdos — CHANGES_REQUESTED** [review]

File: src/main/resources/tabcontrol.mixins.json (hunk 1 above)
Comment (BoogieMonster1O1) [reply src/main/resources/tabcontrol.mixins.json:7]: no

**Comment by BoogieMonster1O1** [comment]
> we already have a config mod. use it :) it avoids so much of that config code you put in

dfu good and best
the config code would have existed anyway

File: src/main/resources/tabcontrol.mixins.json (hunk 1 above)
Comment (BoogieMonster1O1) [reply src/main/resources/tabcontrol.mixins.json:7]: :concern:

**Comment by BoogieMonster1O1** [comment]
this pr be like **merge me please**

**Pull request merged**